.gitignore

```diff
@@ -1,2 +1,3 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+src/*.js
```

Comment:
This is just for any files I create while I'm doing a small test. If I compile a typescript file in that directory, it won't be commited